src/styles/main.scss

```diff
@@ -1,3 +1,247 @@
+$main-accent: #060B35;
+$blue-accent: #00ACDC;
+$secondary: #616070;
+$white: #FFF;
+$steel-blue: #34568b;
+$gray-element: #F3F3F3;
+$white-star: url(../images/star.svg);
+$yellow-star: url(../images/star-active.svg);
+$card-width: 200px;
+$transition: 300ms;
+
+
 body {
   margin: 0;
+
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 400;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: calc(100% - 100px);
+  padding: 0 50px;
+
+  &__logo {
+    margin: 0;
+    padding: 0;
+    border: none;
+    vertical-align: middle;
+  }
+
+  &__navigation {
+    display: flex;
+    height: 60px;
+  }
+}
+
+.menu {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding-inline-start: 0;
+  font-weight: 500;
+  text-transform: uppercase;
+  list-style-type: none;
+
+  margin: 0;
+
+  &__list {
+    height: 100%;
+  }
+
+  &__list:not(:first-child) {
+    margin-left: 20px;
+  }
+
+  &__link {
+    position: relative;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    text-decoration: none;
+    color: $main-accent;
+    height: 100%;
+    box-sizing: border-box;
+
+    &:hover {
+      color: $blue-accent;
+      transition: 300ms;
+    }
+  }
+}
+
+.is-active {
+  color: $blue-accent;
+  transition: $transition;
+}
+
+.is-active::after {
+  content: '';
+  position: absolute;
+  bottom: 0;
+  left: 50%;
+  transform: translateX(-50%);
+  width: 100%;
+  height: 4px;
+  border-radius: 2px;
+  background-color: $blue-accent;
+}
+
+.goods {
+  display: grid;
+  grid-template-columns: repeat(1, $card-width);
+  grid-gap: 46px 48px;
+  justify-content: center;
+  padding: 50px 40px;
+
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(4, $card-width);
+  }
+
+  @media (min-width: 768px) and (max-width: 1023px) {
+    grid-template-columns: repeat(3, $card-width);
+  }
+
+  @media (min-width: 488px) and (max-width: 767px) {
+    grid-template-columns: repeat(2, $card-width);
+  }
+}
+
+.product {
+  width: $card-width;
+  box-sizing: border-box;
+  border: 1px solid $gray-element;
+  border-radius: 5px;
+  padding: 16px;
+
+  transition: transform $transition;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__image {
+    display: block;
+    margin-inline: auto;
+    margin-top: 16px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    color: $main-accent;
+    margin-block: 0;
+    margin-top: 40px;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+
+    &:hover {
+      color: $steel-blue;
+    }
+  }
+
+  &__code {
+    color: $secondary;
+    margin-block: 0;
```

Comment:
Check all elements from devtools and remove redundants (disable property and if nothing happens it means this property is redundant) Fix all cases
```suggestion
```
![image](https://github.com/mate-academy/layout_catalog/assets/70009205/6206f3b2-c5b3-4ac6-94ee-9041fb0b4143)